consul/README.md

```diff
@@ -96,25 +73,62 @@ Reload the Consul Agent to start sending more Consul metrics to DogStatsD.
             source: consul
             service: myservice
     ```
+
     Change the `path` and `service` parameter values and configure them for your environment.
     See the [sample consul.d/conf.yaml][5] for all available configuration options.
 
 3. [Restart the Agent][6].
 
+#### Containerized
+
+For containerized environments, see the [Autodiscovery Integration Templates][7] for guidance on applying the parameters below.
+
+##### Metric collection
+
+| Parameter            | Value                              |
+|----------------------|------------------------------------|
+| `<INTEGRATION_NAME>` | `consul`                           |
+| `<INIT_CONFIG>`      | blank or `{}`                      |
+| `<INSTANCE_CONFIG>`  | `{"url": "https://%%host%%:8500"}` |
+
+##### Log collection
+
+**Available for Agent v6.5+**
+
+Collecting logs is disabled by default in the Datadog Agent. To enable it, see [Docker log collection][8].
+
+| Parameter      | Value                                               |
+|----------------|-----------------------------------------------------|
+| `<LOG_CONFIG>` | `{"source": "consul", "service": "<SERVICE_NAME>"}` |
+
+#### DogStatsD
+
+Alternatively, you can configure consul to send its data to the Agent through [DogStatsD][3] instead of relying on the Agent to pull the data from consul. To achieve this, add your `dogstatsd_addr` nested under the top-level `telemetry` key in the main Consul configuration file:
```

Comment:
```suggestion
Alternatively, you can configure Consul to send data to the Agent through [DogStatsD][3] instead of relying on the Agent to pull the data from Consul. To achieve this, add your `dogstatsd_addr` nested under the top-level `telemetry` key in the main Consul configuration file:
```